job service: get jobs error: value out of range

Error found when the number of tasks is greater than 256
```
2021-04-13T10:11:31Z [ERROR] [/jobservice/api/handler.go:288]: Serve http request 'GET /api/v1/jobs' error: 500 {"code":10014,"message":"failed to get scheduled jobs, q={}","details":"strconv.ParseUint: parsing \"928\": value out of range"}

```

--- a/src/jobservice/mgt/manager.go
+++ b/src/jobservice/mgt/manager.go
@@ -143,7 +143,7 @@ func (bm *basicManager) GetJobs(q *query.Parameter) ([]*job.Stats, int64, error)
 		return nil, 0, errors.New("malform scan results")
 	}
 
-	nextCur, err := strconv.ParseUint(string(values[0].([]byte)), 10, 8)
+	nextCur, err := strconv.ParseInt(string(values[0].([]byte)), 10, 64)
 	if err != nil {
 		return nil, 0, err
 	}
@@ -166,7 +166,7 @@ func (bm *basicManager) GetJobs(q *query.Parameter) ([]*job.Stats, int64, error)
 		}
 	}
 
-	return results, int64(nextCur), nil
+	return results, nextCur, nil
 }
 
 // GetPeriodicExecution is implementation of Manager.GetPeriodicExecution

